ci/common.sh

```diff
@@ -170,7 +170,7 @@ GRPC_SERVER_EXAMPLE_VER=20210819
 
 linux_get_dependencies () {
     apt update
-    apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl libpcre3 libpcre3-dev libldap2-dev
+    apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl libpcre3 libpcre3-dev libldap2-dev gawk
```

Comment:
is this needed?

Reply:
when I test this pr locally, this is needed.

Reply:
that doesn't explain this change 🤔

Reply:
I do this according to the Makefile
https://github.com/apache/apisix/blob/869d095153df75fbe6e3edcf631340965fdd04ab/Makefile#L117-L123

Reply:
please revert this change